### PR TITLE
Automation

### PR DESCRIPTION
**Pull request by kalpanaTech**
Hi,
Please find the "Add Languages and Add Skill" automation code for the review. I'm looking forward to some advices to improve my code. I'll appreciate your feedback.
Thanks

### CODE REVIEW

File: ProjectMars/Feature/Profile.feature [hunk 1]
```diff
@@ -0,0 +1,299 @@
+﻿Feature: ProfileFeature
+
+This is a platform where users can add and show skills and languages ​​known by users.
```
Comment (amanbansalnz) [inline ProjectMars/Feature/Profile.feature]: can you please split your feature files into features,  language and skills! do this with steps definitions too

File: ProjectMars/SpecflowPages/Helpers/CommonMethods.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectMars.SpecflowPages.Helpers
+{
+    internal class CommonMethods
+    {
```
Comment (amanbansalnz) [inline ProjectMars/SpecflowPages/Helpers/CommonMethods.cs]: no required if its empty

File: ProjectMars/SpecflowPages/Helpers/ConstantHelpers.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectMars.SpecflowPages.Helpers
+{
+    internal class ConstantHelpers
```
Comment (amanbansalnz) [inline ProjectMars/SpecflowPages/Helpers/ConstantHelpers.cs]: no required if its empty

File: ProjectMars/SpecflowPages/Utils/Start.cs [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+            //Login page object initialization and definition
+            SignIn signInObj = new SignIn();
+            signInObj.LoginActions(driver);
+        }
+        [Test, Order(1), Description("This test will be verified the Language Adding process")]
+        
+        public void AddLanguage_Start()
+        {
+            AddLanguage addNewLanguageButtonActionsObj = new AddLanguage();
```
Comment (amanbansalnz) [inline ProjectMars/SpecflowPages/Utils/Start.cs]: why do you have this?

Comment (amanbansalnz) [reply ProjectMars/SpecflowPages/Utils/Start.cs]: you are missing step definitions please this for specflow exmaple https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html

File: ProjectMars/Feature/Profile.feature [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+People looking for skills and languages ​​can see what details other users have.
+User can perform actions add, Update, delete etc.
+
+
+Background:
+	Given I logged into profile page successfully
+	
+@regression
+Scenario Outline: User add languages
+	When I click on the Add New button
```
Comment (amanbansalnz) [inline ProjectMars/Feature/Profile.feature]: i see your do cleanup after each test, but have you cleaned up all data before any test has started?

**Review by amanbansalnz — CHANGES_REQUESTED** [review]
Hey, I have reviewed your task please see the pr for the suggested changes to make

**Comment by kalpanaTech** [comment]
Hi Aman, Thank you for your review. I have updated the code based on your suggestions. Could you please take a look?

I previously added the Start.cs page, thinking it was necessary for SpecFlow, but it wasn't needed, so I removed it along with other unnecessary files.
I added a new step to clean up all data before the tests start.
I combined the steps for adding duplicate data into one, instead of two separate steps.

Thanks

**Pull request merged**